src/k2/tso/service/Service.cpp

```diff
@@ -158,12 +158,6 @@ seastar::future<bool> TSOService::_assign(uint64_t tsoID, k2::Duration errBound)
             K2LOG_I(log::tsoserver, "Error bound validation succeeded? {}", isValidErrorBound);

```

Comment:
Also, remove the _collectWorkerURLs method and usage just above, as well as the _workersURLs member var

Reply:
Removed.